### PR TITLE
configstore: Implement GetBytes() method

### PR DESCRIPTION
**Pull request by lpereira**
Similar to the Get() method, this one will return the byte slice as is from the Dictionary, without converting it to a string.  May be useful in cases where configuration is stored as binary blobs.

### CODE REVIEW

File: configstore/configstore.go [hunk 1]
```diff
@@ -86,26 +86,36 @@ func (s *Store) Has(key string) (bool, error) {
...
 }
+
+// Get returns the item in the config store with the given key.
+func (s *Store) Get(key string) (string, error) {
+	buf, err := s.GetBytes(key)
+	if err != nil {
+		return "", err
+	}
+
+	return string(buf), nil
```
Comment (ydnar) [inline configstore/configstore.go:120]: Does this optimize away to the non-allocating byte slice to string conversion?

File: internal/abi/fastly/hostcalls_noguest.go [hunk 2]
```diff
@@ -297,6 +297,10 @@ func OpenDictionary(name string) (*Dictionary, error) {
 	return nil, fmt.Errorf("not implemented")
 }
 
+func (d *Dictionary) GetBytes(key string) ([]byte, error) {
+	return []byte{}, fmt.Errorf("not implemented")
```
Comment (ydnar) [inline internal/abi/fastly/hostcalls_noguest.go:301]: Why not `return nil, ...`?

Second, `fmt.Errorf("const string")` should be replaced with a declared `errors.New("...")` in a global or a custom `type errorString string` and a const.

File: configstore/configstore.go (hunk 1 above)
Comment (lpereira) [reply configstore/configstore.go:120]: I have patches to TinyGo that will do that, yes.

File: internal/abi/fastly/hostcalls_noguest.go (hunk 2 above)
Comment (cee-dub) [reply internal/abi/fastly/hostcalls_noguest.go:301]: This is all over the stubbed implementations without `wasip1` build tags.

Comment (lpereira) [reply internal/abi/fastly/hostcalls_noguest.go:301]: Sure, I can return `nil`.  (The rest here I just copied what was already there. I didn't want to make changes that weren't related to the change in this PR.)

Comment (lpereira) [reply internal/abi/fastly/hostcalls_noguest.go]: (Patch to return `nil` pushed.)

File: configstore/configstore.go (hunk 1 above)
Comment (cee-dub) [reply configstore/configstore.go:120]: I added this to the integration test, and see the following output:
`~/go/src/github.com/fastly/compute-sdk-go/integration_tests/config_store/main_test.go:47:23: object allocated on the heap: escapes at line 52`
``` go
	tb, err := d.GetBytes("twitter")
	if err != nil {
		t.Fatal(err)
	}
	if got, want := tb, "https://twitter.com/fastly"; !bytes.Equal(tb, []byte(want)) {
		t.Errorf("Body = %q, want %q", got, want) // line 52
	}
```

Comment (cee-dub) [reply configstore/configstore.go:120]: Can push up that commit to test the new func

**Comment by cee-dub** [comment]
See #115

**Pull request closed without merging**